network-api/networkapi/wagtailpages/factory/__init__.py

```diff
@@ -28,6 +29,7 @@ def generate(seed):
     participate_page_featured_highlights.generate(seed)
     styleguide.generate(seed)
     youtube_regrets_page.generate(seed)
+    publication.generate(seed)
```

Comment:
same here, unless `publication` relies on any of the builds preceding it, let's keep it alphabetically sorted.